neo4j/README.md

```diff
@@ -26,9 +29,11 @@ To install the neo4j check on your host:
 
 1. Edit the `neo4j.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your neo4j performance data. See the [sample neo4j.d/conf.yaml][3] for all available configuration options.
 
-2. The neo4j_url has been replaced by host. Please ensure any updates use host.
+2. Edit the `neo4j.d/neo4j.yaml` file in the `conf.d/` folder at the root of your Agent's configuraiton directory. See the [sample neo4j.d/neo4j.yaml][10] for all available configuration options.
 
-3. [Restart the Agent][4].
+3. The neo4j_url has been replaced by host. Please ensure any updates use host.
```

Comment:
Is this applicable to both versions?

Reply:
Removing since it is legacy from the 3.5 -> 4.0 code